bot.ts

```diff
@@ -29,7 +35,40 @@ const noop = () => void 0;
 // Gathered all the output messages in single place for easier translation.
 //---------------------------------------------------------------------
 
-const MsgType = {
+enum MsgType {
+	MISSING_ACTION = 0,
+	DATA_PARSING_ERROR = 1,
+	MISSING_ACTIONS = 2,
+	DUPLICATE_SLASH_COMMAND = 3,
+	INVALID_SLASH_NAME = 4,
+	DUPLICATE_USER_COMMAND = 5,
+	DUPLICATE_MESSAGE_COMMAND = 6,
+	DUPLICATE_SLASH_PARAMETER = 7,
+	INVALID_SLASH_PARAMETER_NAME = 8,
+	INVALID_SLASH_COMMAND_SERVER_ID = 9,
+	DUPLICATE_BUTTON_ID = 10,
+	DUPLICATE_SELECT_ID = 11,
+	TOO_MANY_SPACES_SLASH_NAME = 12,
+	SUB_COMMAND_ALREADY_EXISTS = 13,
+	SUB_COMMAND_GROUP_ALREADY_EXISTS = 14,
+
+	MISSING_APPLICATION_COMMAND_ACCESS = 100,
+	MISSING_MUSIC_MODULES = 101,
+
+	MUTABLE_VOLUME_DISABLED = 200,
+	MUTABLE_VOLUME_NOT_IN_CHANNEL = 201,
+	ERROR_GETTING_YT_INFO = 202,
+	ERROR_CREATING_AUDIO = 203,
+
+	MISSING_MEMBER_INTENT_FIND_USER_ID = 300,
+	CANNOT_FIND_USER_BY_ID = 301,
+
+	SERVER_MESSAGE_INTENT_REQUIRED = 400,
+	CHANNEL_PARTIAL_REQUIRED = 401,
+
+}
```

Comment:
If this change gets approved then I'll delete the commented out portion :) or restore it if denied 🙃